artemis-server/src/main/java/org/apache/activemq/artemis/core/persistence/impl/journal/JournalStorageManager.java

```diff
@@ -461,8 +462,7 @@ void deleteLargeMessageFile(final LargeServerMessage largeServerMessage) throws
          try {
             if (isReplicated() && replicator.isSynchronizing()) {
                synchronized (largeMessagesToDelete) {
```

Comment:
If it's now using CHM there is no need to sync on it

Reply:
@franz1981 looking at the code, looks like its just vanilla HM 

   protected final Map<Long, LargeServerMessage> largeMessagesToDelete = new HashMap<>();